README.rst

```diff
@@ -56,6 +56,18 @@ This plugin also adds a new pytest option named
 when molecule drivers are not loading. Current default is ``None`` but you
 can choose marks like ``skip`` or ``xfail``.
 
+Using pytest markers
+--------------------
+
+If you want to add various pytest markers like ``xfail`` or ``xskip``,
+just add something like below to your ``molecule.yml`` file:
+
+.. code-block:: yaml
+
+    tags:
```

Comment:
Is this name appropriate? I have a feeling we might confuse users here since Ansible also uses tags and those two concepts are not related at all. Maybe we should go with `marks` since this is the name pytest uses?

Reply:
+1 markers sounds more appropriate. Otherwise, this will cause a lot of confusion.